core/extension/gdextension_interface.cpp

```diff
@@ -1048,6 +1048,25 @@ static GDExtensionScriptInstancePtr gdextension_script_instance_create(const GDE
 	return reinterpret_cast<GDExtensionScriptInstancePtr>(script_instance_extension);
 }
 
+static GDExtensionScriptInstancePtr gdextension_object_get_script_instance(GDExtensionConstObjectPtr p_object, GDExtensionConstObjectPtr p_language) {
+	if (!p_object || !p_language) {
+		return nullptr;
+	}
+
+	const Object *o = (const Object *)p_object;
+	ScriptInstanceExtension* script_instance_extension = reinterpret_cast<ScriptInstanceExtension *>(o->get_script_instance());
```

Comment:
```suggestion
	ScriptInstanceExtension *script_instance_extension = reinterpret_cast<ScriptInstanceExtension *>(o->get_script_instance());
```
[Code style](https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html), I'd suggest using `clang-format`, instructions in the same link